independent-publisher-connectors/Finnish BIS/apiDefinition.swagger.json

```diff
@@ -104,7 +104,7 @@
                               "type": "integer",
                               "format": "int32"
                             },
-                            "name": {
+                            "description": {
```

Comment:
@tpertila can you explain this one too please? How will this affect the current users?

Reply:
There is no such a column as name in liquiditions but description column instead. If the query result contains the description column (in liquidations), it fails. Because there is no definition for the column.

So after the change users get results instead of an error message when searching companies having that liquidition information on the resultset.